labelme/app.py

```diff
@@ -1852,9 +1855,7 @@ def openPrevImg(self, _value=False):
 
     def openNextImg(self, _value=False, load=True):
         keep_prev = self._config["keep_prev"]
-        if QtWidgets.QApplication.keyboardModifiers() == (
-            Qt.ControlModifier | Qt.ShiftModifier
-        ):
+        if shift_and_ctrl_pressed():
```

Comment:
unnecessary change?

Reply:
No - just using the same new function for the same functionality.

Reply:
i mean, this pr is for deleting poly right? why you change keep_prev?

Reply:
It was a bit of a side effect – I think, back last year when I originally made this PR, I figured it'd be worth it to unify the modifier key functionality used both for "hold shift to delete polys" and "hold shift+ctrl" as used here, into canonical helper functions.